plugin/central.vim

```diff
@@ -3,16 +3,43 @@
 " Version: 0.2.0
 " License: BSD
 
-set backupdir=~/.vim/backup//
-set directory=~/.vim/swap//
-set undodir=~/.vim/undo//
+set backupdir=$VIMHOME/backup//
+set directory=$VIMHOME/swap//
+set undodir=$VIMHOME/undo//
 
-if !isdirectory($HOME . "/.vim/backup")
-    call mkdir($HOME . "/.vim/backup", "p")
+for dir in [ &backupdir, &directory, &undodir ]
+    if !isdirectory(dir)
+        call mkdir(dir, 'p')
```

Comment:
so here with 

`call mkdir(dir, 'p')` This surpasses any error messages where the directory may already exist, right? 

Awesome stuff, Matt.

Reply:
ahhh, very nice! thanks for explaining that